finalize/finalize.gm9

```diff
@@ -23,8 +23,19 @@ if find "0:/Nintendo 3DS/$[SYSID0]/????????????????????????????????_user-id1" NU
 	set PREVIEW_MODE "0:/finalize/img/error18.png"
 	echo "Error #18: MSET9 detected\n \nMSET9 is still installed on this console.\nYou MUST remove it before continuing.\nScan the QR code above for instructions.\n \nIf this error persists, ask for help\non Discord: https://discord.gg/MWxPgEp"
 	poweroff
+
+# What is the NULL parameter here for? If I changed it, would it become a variable holding the name for the found folder?
+# Where can I read up on gm9script?
+# It was here https://github.com/eiiiiix/EixMode9/blob/master/HelloScript.gm9
+# And I was right.
+else if find "0:/Nintendo 3DS/$[SYSID0]/????????????????????????sdmc??b9 MSET9ID1 # Else if because I don't want the user to power on and have the system create another ID1
+	# Silently delete it.
+	rm -s $[MSET9ID1]
+	# -o = optional, i.e. don't sh*t itself when it fails. I don't want this to fail.
```

Comment:
if it does fail, it should ideally be surrounded by an if clause to give the user more info rather than gm9's "failed lmao"

Reply:
oh right by the way the quotation mark isn't closed on the if find line

Reply:
About this.... how do I explain to the user that the hax ID1 didn't get deleted exactly

Reply:
most likely something along the lines of "Failed to delete hax ID1. This shouldn't happen. Ask for help [...]"

I am wondering whether GM9 is able to handle this through scripting though -- GM9 definitely can delete hax id1 in general but it seems like what's currently written here doesn't work (even though it should)

On another note since this probably falls under gm9's protection scheme you most likely would need to prompt the user to allow key combos (so something like "hax id1 detected, please do the key combo on the next prompt")